client/src/components/SystemStatus/SystemStatus.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { useEffect } from 'react';
+import { Route, Switch, Link, useHistory, useLocation } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { LoadingSpinner, Section, SectionTableWrapper } from '_common';
+import * as ROUTES from '../../constants/routes';
+import { Sidebar, SectionMessage } from '_common';
+import Sysmon from '../SystemMonitor';
+import { SystemStatusQueueTable } from './SystemStatusQueueTable/SystemStatusQueueTable';
+
+import './SystemStatus.global.css';
+import styles from './SystemStatus.module.scss';
+
+const root = `${ROUTES.WORKBENCH}${ROUTES.SYSTEM_STATUS}`;
+
+const SystemStatusSidebar = ({ systemList }) => {
+  var sidebarItems = [];
+
+  systemList.forEach((system) => {
+    sidebarItems.push({
+      to: `${root}/${system.hostname}`,
+      label: `${system.display_name}`,
+      disabled: false,
+      hidden: false,
+      iconName: '',
+    });
+  });
+
+  return <Sidebar sidebarItems={sidebarItems} />;
+};
+
+const Layout = ({ hostname }) => {
+  const {
+    loading,
+    error: loadingError,
+    list: systemList,
+  } = useSelector((state) => state.systemMonitor);
+
+  const selectedSystem = systemList.find((sys) => sys.hostname === hostname);
+
+  if (loadingError) {
+    return (
+      <Section className={styles['no-results-message']}>
+        <SectionMessage type="info">
+          Unable to gather system information
+        </SectionMessage>
+      </Section>
+    );
+  }
+
+  return (
+    <>
+      <Section
+        className={styles['section']}
+        contentLayoutName="oneColumn"
+        content={
+          <>
+            <SectionTableWrapper
+              className={styles['sysmon-content']}
+              contentShouldScroll
+            >
+              <Sysmon system={selectedSystem?.hostname}></Sysmon>
+            </SectionTableWrapper>
```

Comment:
**UI Problem**
I noticed this table can have a scrollbar. It should not.

<details><summary>Screenshot</summary>

![core-portal 807 screenshot scrollbar](https://github.com/TACC/Core-Portal/assets/62723358/65862f25-0ad3-45e6-a157-d8320e32f7d6)

</details>

1. Did you notice this?
2. If so, what did you try to get rid of it?

<sup>I would investigate a solution myself, but [I am having trouble setting up my local environment](https://tacc-team.slack.com/archives/GQW4Q8HLG/p1687295165655549). If you let me know what hasn't worked, I can give you specific options.</sup>

Reply:
I removed the scrollbar in #816.